FreeRTOS/FreeRTOS-Kernel
# Update RISC-V IAR port to support vector mode.

Description
-----------
Add support for vectored interrupt mode for the RISCV IAR port.

Test Steps
-----------
Integrated the changes into the `RISC-V_RV32_SiFive_HiFive1-RevB_IAR` demo and ran the kernel port `main_full` test. It's  been running for 3 days and not reported an issue.

Related Issue
-----------
n/a


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
